notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/BagMiddleware.java

```diff
@@ -0,0 +1,48 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Does no routing of Middleware based on conditions, simply applies all enrichments it is aware of
+ * then calls the next Middleware.
+ */
+class BagMiddleware implements InstallationMiddleware {
+
+    private final Set<InstallationEnricher> enrichers;

```

Comment:
Let's be consistent with our member variables, prepending `m` for example `mEnrichers`